### PR TITLE
Templates created and added

### PR DESCRIPTION
**Pull request by cypher-naman**
static folder with no file was not able to get added in the commit.

### CODE REVIEW

**Comment by gutsytechster** [comment]
@sharma-naman Have you added these changes in other branch?

**Comment by GeekyShacklebolt** [comment]
Hi @sharma-naman, @VirtualRcoder!
You could give proper commit messages. It must be precise and should reflect what changes you are making in a commit. Try using a commit message standard. Refer [this](https://github.com/CuriousLearner/dotfiles/blob/master/.gitmessage)

File: Feedback/templates/Feedback/index_col.html [hunk 1]
```diff
@@ -0,0 +1,721 @@
...
+  <head>
+      <meta charset="utf-8">
+      <title>College Work</title>
+      <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+      <style>
+      body{background: url(http://digitallearning.eletsonline.com/wp-content/uploads/2018/04/Mettl-partners-with-NSDC-to-innovate-assessment-technology.jpg);
+         background-size:cover;
+         background-repeat:no-repeat;
+         opacity: 0.7;
+       padding: 30px 30px 0px 30px;}
```
Comment (GeekyShacklebolt) [inline Feedback/templates/Feedback/index_col.html]: Please correct indentation.

File: Feedback/templates/Feedback/index_col.html [hunk 2]
```diff
@@ -0,0 +1,721 @@
...
+
+       .container{ background: #2e7d32}
+           .landscape {
+        height: 130px;
```
Comment (GeekyShacklebolt) [inline Feedback/templates/Feedback/index_col.html]: Correct indentation.

File: Feedback/templates/Feedback/index_col.html [hunk 3]
```diff
@@ -0,0 +1,721 @@
...
+        width: 1500px;
+            }
+
+            img {
+    max-width: 100%;
```
Comment (GeekyShacklebolt) [inline Feedback/templates/Feedback/index_col.html]: Correct indentation.

File: Feedback/templates/Feedback/index_col.html (hunk 1 above)
Comment (sharma-naman) [reply Feedback/templates/Feedback/index_col.html]: Ok sure will do asap, Thanks for the suggestions.

**Comment by VirtualRcoder** [comment]
No more required

**Pull request closed without merging**